webapp/src/components/TransactionsHistory.js

```diff
@@ -1,42 +1,111 @@
 /* eslint camelcase: 0 */
-import React, { memo } from 'react'
+import React, { memo, useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 import Card from '@material-ui/core/Card'
 import Grid from '@material-ui/core/Grid'
 import CardContent from '@material-ui/core/CardContent'
 import Typography from '@material-ui/core/Typography'
 import LinearProgress from '@material-ui/core/LinearProgress'
+import LaunchIcon from '@material-ui/icons/Launch'
 import { useSubscription } from '@apollo/react-hooks'
 
 import { BLOCK_TRANSACTIONS_HISTORY } from '../gql'
 import { formatWithThousandSeparator, getBlockNumUrl } from '../utils'
+import { generalConfig } from '../config'
 
-const BodyGraphValue = ({ loading, value }) => {
+const BodyGraphValue = ({ loading, value, classes, href }) => {
   if (loading) return <LinearProgress />
 
-  if (typeof value !== 'number') {
-    return <>{value}</>
-  }
-
   return (
     <Typography component="p" variant="h6">
       {value}
+      {href && (
+        <a href={href} target="_blank" rel="noopener noreferrer">
+          <LaunchIcon className={classes.svgLink} color="primary" />
+        </a>
+      )}
     </Typography>
   )
 }
 
 BodyGraphValue.propTypes = {
   loading: PropTypes.bool,
-  value: PropTypes.any
+  value: PropTypes.any,
+  classes: PropTypes.object,
+  href: PropTypes.string
 }
 
 BodyGraphValue.defaultProps = {
   value: 0,
-  loading: false
+  loading: false,
+  classes: {}
 }
 
 const TransactionsHistory = ({ t, classes }) => {
   const { data, loading } = useSubscription(BLOCK_TRANSACTIONS_HISTORY)
+  const [allTimeHigh, setallTimeHigh] = useState({})
+
+  useEffect(() => {
+    if (data?.stats?.[0]?.tps_all_time_high?.blocks?.length) {
+      const result = data.stats[0].tps_all_time_high.blocks.reduce(
+        (prev, current) => {
+          return {
+            tps: {
+              value:
+                prev.value > current.transactions_count
+                  ? prev.value
+                  : current.transactions_count,
+              link: getBlockNumUrl(
+                prev.value > current.transactions_count
+                  ? prev.link
+                  : current.block_num
+              )
+            },
+            cpu: {
+              value: formatWithThousandSeparator(
+                prev.value > current.cpu_usage_percent
```

Comment:
@tetogomez why are you checking if current value if higher than previous? It already should be done at the backend service

Reply:
@adriexnet because we just need to print one value instead two and the blocks array have more than one object